src/UnitySampleProject/Assets/Samples/web3.unity SDK/2.5.0/Web3.Unity Samples/Scenes/SampleMain.unity

```diff
@@ -15427,8 +15639,8 @@ MonoBehaviour:
   m_Script: {fileID: 11500000, guid: 9a86b695c20675b43ba7a0f0a496749c, type: 3}
   m_Name: 
   m_EditorClassIdentifier: 
-  apiSecretKey: Fill In Your API Secret Key From Storage
-  bucketId: Fill In Your Bucket ID From Storage
+  apiSecretKey: eyJhbGciOiJFUzI1NiIsInR5cCI6IkpXVCJ9.eyJpYXQiOjE3MTY0MDA1MzEsImNuZiI6eyJqa3UiOiIvY2VydHMiLCJraWQiOiI5aHE4bnlVUWdMb29ER2l6VnI5SEJtOFIxVEwxS0JKSFlNRUtTRXh4eGtLcCJ9LCJ0eXBlIjoiYXBpX3NlY3JldCIsImlkIjo5NDc2LCJ1dWlkIjoiMmEyYWY0MWYtMTQ4MC00YWE0LWE1MDYtMjJkNjA3OTUzNjU4IiwicGVybSI6eyJiaWxsaW5nIjoiKiIsInNlYXJjaCI6IioiLCJzdG9yYWdlIjoiKiIsInVzZXIiOiIqIn0sImFwaV9rZXkiOiJWTUFLUEZXSlpPSlBZSE5LSllYTSIsInNlcnZpY2UiOiJzdG9yYWdlIiwicHJvdmlkZXIiOiIifQ.he_5VYdwIItgAICTLT5mxAxJmY--LPAEyVAGGZfy2YW8h4aYtVcZy5XIDm0gAC6bYQYC1K4aqLstNYOF9hNVRQ
```

Comment:
I believe the values here need to be removed

Reply:
good thing i clear my keys before upload xD